oci/oathkeeper/image.yaml

```diff
@@ -0,0 +1,16 @@
+version: 1
+
+release:
+  latest:
+    end-of-life: "2023-06-22T00:00:00Z"
+    edge: 0.40.3
```

Comment:
this needs to be either a revision number or a fully-formed channel. So I think you're missing the `_edge` if your goal is to follow the 0.40.3_edge tag you're building